### PR TITLE
Pass the list of volume for mayaapiserver

### PR DESCRIPTION
**Pull request by inyee786**
**What this PR does / why we need it**:
pass the list of current volume to mayaapiserver to filter the cas data

Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
can't paste the output as it will be available when it present in the real cluster and openebs is present

### CODE REVIEW

File: src/app/workloads/workloads.component.ts [hunk 1]
```diff
@@ -279,6 +281,7 @@ export class WorkloadsComponent implements OnInit, OnDestroy {
       .getPodDetails(this.currentRoute[1], this.currentRoute[1])
       .subscribe(res => {
         this.litmuspod = res.statefulSet;
+        this.mayapvc = res.pvc.pvc
```
Comment (chandankumar4) [inline src/app/workloads/workloads.component.ts]: Why this two times pvc used?

File: src/app/model/data.model.ts [hunk 2]
```diff
@@ -147,17 +151,15 @@ export interface overAllStatus {
       dockerImage: string;
     }
   ];
-  pvc: [
+  pvc: 
     {
       numberofPvc: number;
```
Comment (chandankumar4) [inline src/app/model/data.model.ts]: numberofPvc -> numberOfPVC

File: src/app/model/data.model.ts [hunk 3]
```diff
@@ -85,12 +85,16 @@ export interface applicationPod {
 }
 export interface pvc {
   numberofPvc: number;
```
Comment (chandankumar4) [inline src/app/model/data.model.ts]: numberofPvc -> numberOfPVC

File: src/app/workloads/workloads.component.ts (hunk 1 above)
Comment (inyee786) [reply src/app/workloads/workloads.component.ts]: pvc is stored inside the pvc
something like this
```
pvc {
  numberofPvc: number;
  pvc: 
    {
      name: string;
      volumeName: string;
    }
}
```

Comment (chandankumar4) [reply src/app/workloads/workloads.component.ts]: change the pcv -> pvcs { pvc {} }

**Pull request merged**